jetbrains-gateway/src/software/aws/toolkits/jetbrains/gateway/welcomescreen/WorkspaceDetails.kt

```diff
@@ -73,7 +73,8 @@ class WorkspaceDetails(ws: Workspace, workspaces: WorkspaceList, cawsClient: Cod
             ws.status == DevEnvironmentStatus.FAILED -> JBLabel(AllIcons.General.Error)
             else -> JBLabel(AwsGatewayIcons.GATEWAY_STOPPED)
         }
-        statusIconLabel.toolTipText = ws.status.name
+
+        statusIconLabel.toolTipText = if (ws.status == DevEnvironmentStatus.STOPPED) "${ws.status.name}: ${ws.statusReason}" else ws.status.name
```

Comment:
why not
```suggestion
        statusIconLabel.toolTipText = if (ws.statusReason != null) "${ws.status.name}: ${ws.statusReason}" else ws.status.name
```

Reply:
Agreed, will make changes.

Reply:
Actually, setting `ws.statusReasonisNotEmpty()` instead of `ws.statusReason != null` will be working perfectly.